src/act.movement.cpp

```diff
@@ -751,8 +751,9 @@ void move_vehicle(struct char_data *ch, int dir)
     ch->in_room = was_in;
   else
     ch->in_room = NULL;
-
-  if ((get_speed(veh) > 80 && SECT(veh->in_room) == SPIRIT_CITY) || veh->in_room->icesheet[0])
+    
+  // Don't do crash tests on autonav.
```

Comment:
I'd argue that if autonav is moving at maximum speed, it should be doing crash tests against some standardized skill rating. A car going 400+ kph over a sheet of ice is going to have a bad day regardless of who's at the wheel.